frontend/src/features/crawl-workflows/new-workflow-dialog.ts

```diff
@@ -12,78 +13,25 @@ export type SelectJobTypeEvent = CustomEvent<"url-list" | "seed-crawl">;
  */
 @localized()
 @customElement("btrix-new-workflow-dialog")
-export class NewWorkflowDialog extends LitElement {
-  // postcss-lit-disable-next-line
-  static styles = css`
-    .title,
-    .container {
-      margin: var(--sl-spacing-large) 0;
-    }
-
-    .container {
-      display: flex;
-      flex-wrap: wrap;
-      gap: var(--sl-spacing-4x-large);
-      justify-content: center;
-    }
-
-    .heading {
-      font-size: var(--sl-font-size-large);
-      font-weight: var(--sl-font-weight-semibold);
-      margin-top: 0;
-      margin-bottom: var(--sl-spacing-small);
-      line-height: 1;
-    }
-
-    .description {
-      color: var(--sl-color-neutral-500);
-      margin: 0;
-    }
-
-    .jobTypeButton {
-      padding: 0.25rem;
-      display: block;
-      width: 16.5rem;
-      cursor: pointer;
-      background: none;
-      text-align: left;
-      border: none;
-      border-radius: 0.75rem;
-    }
-
-    figure {
-      margin: 0;
-      padding: 0;
-    }
-
-    .jobTypeButton:hover .jobTypeImg {
-      transform: scale(1.05);
-    }
-
-    .jobTypeImg {
-      width: 100%;
-      max-height: 9rem;
-      transition-property: transform;
-      transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
-      transition-duration: 150ms;
-      margin-bottom: var(--sl-spacing-small);
-    }
-  `;
-
+export class NewWorkflowDialog extends TailwindElement {
   @property({ type: Boolean })
   open = false;
 
   render() {
     return html`
       <btrix-dialog
-        .label=${msg("Choose New Workflow Type")}
+        .label=${msg("Create a New Workflow")}
         .open=${this.open}
         style="--width: 46rem"
       >
-        <div class="container">
+        <p class="mb-7 mt-3 text-base text-neutral-700">
+          ${msg("How do you want to specify which pages to crawl?")}
```

Comment:
This feels a little wordy to me. Would something like "What would you like to crawl?" get across enough info?

Also feel like this should maybe be centered?

Reply:
Wording change is nice! What about just replacing the dialog title?

<img width="747" alt="Screenshot 2024-08-20 at 5 31 22 PM" src="https://github.com/user-attachments/assets/24949861-936d-4ef3-ad32-b5540a7c6832">

Reply:
Oh sure, that looks good!